examples/jsm/tsl/display/AfterImageNode.js

```diff
@@ -1,9 +1,6 @@
 import { RenderTarget, Vector2, QuadMesh, NodeMaterial, RendererUtils, TempNode, NodeUpdateType } from 'three/webgpu';
 import { nodeObject, Fn, float, uv, texture, passTexture, uniform, sign, max, convertToTexture } from 'three/tsl';
 
-const _size = /*@__PURE__*/ new Vector2();
-const _quadMeshComp = /*@__PURE__*/ new QuadMesh();
```

Comment:
Using a quad mesh in module scope is a pattern frequently used in FX classes.

Does the error mentioned in https://github.com/mrdoob/three.js/issues/30832 happen with other classes as well? E.g. `AnamorphicNode`?

Reply:
I just did a quick test and it seems to works fine on anamorphic node

Reply:
@Mugen87  the main difference between `anamorphic` and `afterImage` is one set the material in the nodeBuilder ( afterImage ) , the other one in the updateBefore

Reply:
> one set the material in the nodeBuilder ( afterImage ) 

Do you mind explaining in more detail. I'm afraid I don't understand yet. 

If possible, I would prefer a solution that avoids handling quad meshes differently.

Reply:
`afterImage` : `_quadMeshComp.material = materialComposed` is in `setup( builder )`
So the material is override by the new one when there is another afterImage and it create the bug with multiple afterImage / PostProcessing.

`anamorphic` :`_quadMesh.material = this._material` is set in `updateBefore( frame )` so before the rendering, therefore it allow one _quadMesh to be share between all anamorphic instance with no problem.

The second approach (`anamorphic`) is better and functionnal with multiple node /PostProcessing, but then I'm wondering why there is more than one  _quadMesh for all fx node and why not only one quadMesh share between all of them?

Reply:
Thanks for the clarification! That is indeed a bug. `_quadMeshComp.material = materialComposed;` should be moved to `updateBefore()` instead. Do you mind updating the PR with that fix?

> but then I'm wondering why there is more than one _quadMesh for all fx node and why not only one quadMesh share between all of them?

I don't think that you gain much since all quad meshes already share a single geometry. And you the materials are always assigned so I think maintaining an instance per module is the more straightforward approach.

Reply:
Yes I will update the PR now, but the current approach by module is not very elegant and errors prone like the one in afterImage as the design pattern by module is not classic.
One by class instance, as static property or one for all postprocessing seems a more classical approach.

Reply:
interesting, maybe its time as threejs orient itself to support webgpu ? also looking at [caniuse es2020](https://caniuse.com/?search=es2020) & [es2020 Feature support list](https://caniuse.com/?feats=mdn-javascript_operators_optional_chaining,mdn-javascript_operators_nullish_coalescing,mdn-javascript_builtins_globalthis,es6-module-dynamic-import,bigint,mdn-javascript_builtins_promise_allsettled,mdn-javascript_builtins_string_matchall,mdn-javascript_statements_export_namespace,mdn-javascript_operators_import_meta)  i think it will be safe on new version of threejs to go for it. @mrdoob

Reply:
@Mugen87 i change the fix , using `updateBefore( frame ) ` , tested on my project and it works correctly now with multiple postprocessing / afterImage node

Reply:
Regarding ES2020: The big blocker were build tools like Webpack 4 at that time. Since we don't transpile, must must make sure tools like Webpack 4 are compatible with the raw code base.

Reply:
Understand, just checked and webpack4 it isnt support anymore ( last commit 6years ago ) so I guess its safe to assume users using latest threejs also evolve but i dont have the full scope / policy of threejs and let you guys decide! thanks for validate the PR.

Reply:
Another note: I did a quick analyze using ai on other node and afterImageNode looks like an isolated case ;)